libcxx/test/std/utilities/expected/expected.void/observers/value.lwg3940.verify.cpp

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------------------===//
```

Comment:
let's move this test under libcxx/test/libcxx/[utilities/expected/expected.void
The folder "std" is for tests that should pass on all implementations (not only libc++, and AFAIK, MSVC STL uses our tests under libcxx/test/std)